sbin/initialize_frontier_squid.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if [[ ! -e /etc/squid/squid.conf ]]; then
+  echo "Generating squid.conf..."
+  /etc/squid/customize.sh < /etc/squid/squid.conf.frontierdefault > /etc/squid/squid.conf
+fi
+
+chown -R squid:squid /var/cache/squid
+
+if [[ ! -d /var/cache/squid/00 ]]; then
+  echo "Initializing cache..."
+  /usr/sbin/squid -N -f /etc/squid/squid.conf -z
```

Comment:
I don't know the details of the squid invocation well enough. What does this do? Init the cache and exit?

Reply:
Yes.. BTW: This was in the old
/app/entrypoint.sh
Just moved it out into a separate file that does not also start the daemon.

Reply:
As long as we're not starting two squid processes, that sounds good to me!